packages/core/test/helpers/producers.ts

```diff
@@ -156,6 +157,38 @@ export const scoreResult = factoryMethod<ScoreResult>(() => ({
   transpileErrors: 0
 }));
 
+export const metrics = factoryMethod<Metrics>(() => ({
```

Comment:
Can we move this to the `@stryker-mutator/test-helpers`? It might already be there, because it is also used in the html reporter